MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  MovieDetailsViewController.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 30.03.2023..
+//
+
+import Foundation
+import PureLayout
+import MovieAppData
+
+
+class MovieDetailsViewController : ViewController, UICollectionViewDelegate {
+    private var myImageView: UIImageView!
+    private var ratingLabel: UILabel!
+    private var userScoreLabel: UILabel!
+    private var nameLabel: UILabel!
+    private var yearLabel: UILabel!
+    private var genreLabel: UILabel!
+    private var favouriteButton: UIButton!
+    private var overviewLabel: UILabel!
+    private var overviewText:UITextView!
+    let details = MovieUseCase().getDetails(id: 111161)
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        print(details)
+        buildView()
+        collection()
+    }
+    
+    func imageFunc(){
```

Comment:
Add methods `createViews`, `styleViews` and `defineLayoutForViews`, same as in `LoginViewController`.